lxd/instances_post.go

```diff
@@ -863,12 +862,7 @@ func instancesPost(d *Daemon, r *http.Request) response.Response {
 			return api.StatusErrorf(http.StatusBadRequest, "Target only allowed when clustered")
 		}
 
-		var targetMember, targetGroup string
-		if strings.HasPrefix(target, "@") {
-			targetGroup = strings.TrimPrefix(target, "@")
-		} else {
-			targetMember = target
-		}
+		_, targetGroup := shared.TargetDetect(queryParam(r, "target"))
```

Comment:
The changes affecting instancesPost should be separated from the changes for instancePost, and come earlier in the PR.
This will aid with both reviewing and backporting into LTS branches.